src/test/java/org/gridsuite/study/server/StudyControllerDynamicSimulationTest.java

```diff
@@ -90,11 +102,16 @@ public class StudyControllerDynamicSimulationTest {
 
     private static final int STOP_TIME = 500;
 
+    private static final String VARIANT_ID = "variant_1";
+
+    private static final String CASE_UUID_STRING = "00000000-8cf0-11bd-b23e-10b96e4ef00d";
+    private static final UUID CASE_UUID = UUID.fromString(CASE_UUID_STRING);
+
     private static final String STUDY_UUID_STRING = "00000000-0000-0000-0000-000000000000";
     private static final UUID STUDY_UUID = UUID.fromString(STUDY_UUID_STRING);
 
-    private static final String ROOT_NODE_UUID_STRING = "22222222-0000-0000-0000-000000000000";
-    private static final UUID ROOT_NODE_UUID = UUID.fromString(ROOT_NODE_UUID_STRING);
+    private static final String NETWORK_UUID_STRING = "11111111-0000-0000-0000-000000000000";
```

Comment:
In my opinion, using something in random manner makes the test less predictable. 

Only one case I use a random number is the random port of mock server to avoid conflict to a running server, by docker for example.